packages/rulesets/src/spectral/functions/utils.ts

```diff
@@ -12,6 +12,13 @@ export function isLatestCommonTypesVersionForFile(version: string, fileName: str
   return LATEST_VERSION_BY_COMMON_TYPES_FILENAME.get(fileName) === version.toLowerCase()
 }
 
+const ExtensionResourceFullyQualifiedPathReg = new RegExp(".+/providers/.+/providers/.+$", "gi")
+const ExtensionResourceReg = new RegExp("^/{\\w+}/providers/.+$", "gi")
+
+export function isPathOfExtensionResource(path: string) {
```

Comment:
>isPathOfExtensionResource [](http://example.com/codeflow?start=16&length=25)

can u doublecheck that there isnt another function that does the same thing?

Reply:
We have it defined in the legacy utils file, which I dont think can be used